prow/cmd/peribolos/main.go

```diff
@@ -756,6 +780,15 @@ func configureTeam(client editTeamClient, orgName, teamName string, team org.Tea
 		}
 	}
 
+	if team.Privacy != nil && gt.Privacy != string(*team.Privacy) {
+		patch = true
+		gt.Privacy = string(*team.Privacy)
+
+	} else if team.Privacy == nil && (parent != nil || len(team.Children) > 0) && gt.Privacy != "closed" {
```

Comment:
Should we validate the config and error, instead of just overriding it?

Reply:
Ohh okay. I misread it the first time. 

Yeah, those are some dumb defaults. 👎